docs/governance.md

```diff
@@ -0,0 +1,55 @@
+# Governance
+
+ESTree is an open source project that depends on contributions from the community. Anyone may contribute to the project at any time by submitting code, participating in discussions, making suggestions, or any other contribution they see fit. This document describes how various types of contributors work within the ESTree project.
```

Comment:
I wonder if we should say ESTree is an open source project or something else? It doesn't have code or an OSS license, so maybe it's better to say it's a specification?

Reply:
Can we add a licence to this project? AFAIK licence can be applied on any human work and doesn't have to be tied to the program code. I don't have strong preference on the specific licence type, should we discuss it in a separate issue?

Reply:
> I don't have strong preference on the specific licence type, should we discuss it in a separate issue?

If we do add a license, I believe we need to check what Mozilla used for the original AST. I assume it would be same license that applies to SpiderMonkey itself, but best to ask @dherman.

Reply:
I’d say let’s discuss in a separate issue. I’m definitely in favor of figuring out a license, I’d just prefer to do that in a focused area. 

For this PR I’d still like to answer the question of whether or not we want to call this an open source project or just a specification or something else.

Reply:
Opened an issue:
https://github.com/estree/estree/issues/214

Reply:
> For this PR I’d still like to answer the question of whether or not we want to call this an open source project or just a specification or something else.

In practical terms, what are the pros and cons of calling ESTree an open source project versus a specification?

Reply:
This isn’t really a pro/con discussion, it’s more a “which box does this go in?” discussion.

“Open source” is a squishy enough term that I don’t think anyone would object to using it. I personally feel it’s an inaccurate description given the association of the term with source code, which we don’t really have.

Reply:
Just wanted to make sure I wasn't missing anything. I think "specification" signals the intent of the project more clearly.

Reply:
Okay, I'll update to "specification"